grunttasks/generate-valid-styles.js

```diff
@@ -0,0 +1,41 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+var path = require('path');
+var stripSpecialChars = require('./lib/strip-special-chars');
+
+module.exports = function (grunt) {
+  'use strict';
+   grunt.registerMultiTask('generate-valid-styles', 'Generate the list of valid styles for translation', function () {
+    var validStylesRegex = /style=('|")[\s\S]*?('|")/ig;
```

Comment:
So this is following up from another thread where we decided to extract the entire attribute instead of just the contents.
I think @vladikoff wanted us to do that.